units/provisioning_setup_host.go

```diff
@@ -833,3 +839,53 @@ func parseLsblkOutput(lsblkOutput string) ([]blockDevice, error) {
 
 	return devices.BlockDevices, nil
 }
+
+// GetGithubTokensForTask returns a read-only token for owner/repo associated with the task
+// and a read-only token for each module associated with the task.
+func GetGithubTokensForTask(taskId string, ctx context.Context) (string, []string) {
```

Comment:
Can we add a unit test for this?

Reply:
I don't want to generate actual tokens and I don't think it's worth the effort to mock it. I tested it thoroughly on staging.